docs/source/contributing-guide.md

```diff
@@ -0,0 +1,3 @@
+```{include} ../../CONTRIBUTING.md
+```
+[contributing-guide]: contributing-guide
```

Comment:
Oh nice, so we can see the CONTRIBUTING.md file directly on `conda-build` docs this way! What a great solution to keep everything in one place yet accessible from multiple places!

Reply:
Love it.

Reply:
@beeankha , this is how it's handled in the `conda` repository 😄